packages/@vue/cli/lib/Generator.js

```diff
@@ -181,9 +181,13 @@ module.exports = class Generator {
       'private',
       'description',
       'author',
+      'main',
+      'unpkg',
```

Comment:
We could also inlcude the `jsDelivr` field if we add `unpkg`
```suggestion
      'jsDelivr',
      'unpkg',
```